### DISCUSSION TITLE
Can I use SQLG from Python?

### DISCUSSION THREAD

**Discussion by FRaverta** [pietermartin/sqlg#422] [Q&A] [answered]
I am wondering whether SQLG can be use from Python or not.

**Comment by pietermartin** [comment, marked as answer]
Yes but not natively. To use TinkerPop from outside the JVM you have to go via gremlin server.
There are python drivers that talk to gremlin server.
However all of Sqlg's 'native' features are not available via these drivers. Fairly sure it possible to add these features but as I am not a python developer I have not investigated it. The same goes for javascript.
What will not be available is the api to directly create the schema with non standard features like partitions and indexes and some types.
Also bulk mode will not be available.
So all and all till someone does the work to expose these features via the drivers I would not actually suggest using Sqlg outside the JVM.
